src/test/vitest.global-setup.ts

```diff
@@ -0,0 +1,3 @@
+export const setup = () => {
+  process.env.TZ = 'UTC';
+};
```

Comment:
No, not sure. 😛 I just saw your comment to Coro and thought, ooh, let's fix this for everyone! It's just a pain… but I can revert this change if we want to spend time studying it.

Reply:
I've spent some time studying it, but I don't have a concrete solution yet. The problem is that the test is expecting something that is seemingly impossible, and it only becomes apparent in certain time zones.

I'd prefer to revert this for now, and learn whether the data in production matches what's in the test, because if it does then it's a bug that should be fixed (which could turn into a whole thing).